datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -523,24 +523,6 @@ async fn roundtrip_arithmetic_ops() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
-async fn roundtrip_interval_literal() -> Result<()> {
-    roundtrip(
-        "SELECT g from data where g = arrow_cast(INTERVAL '1 YEAR', 'Interval(YearMonth)')",
```

Comment:
Are you referring to the fact that this would actually error at runtime:
```sql
> select arrow_cast(INTERVAL '1 YEAR', 'Interval(YearMonth)');
This feature is not implemented: Unsupported CAST from Interval(MonthDayNano) to Interval(YearMonth)
```

Reply:
Yup